src/model/ActivityCreationResponse.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2025-present Coinbase Global, Inc.
+ *
+ * This file is generated by Openapi Generator https://github.com/openapitools/openapi-generator
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *  Do not edit the class manually.
+ */
+
+import { ActivityType } from './enums/ActivityType';
+
+export type ActivityCreationResponse = {
```

Comment:
this is probably something we should apply to all the other SDKs as there are quite a few APIs that just return this response, but I also recognize you're generating stuff here and that's probably where these came from right?

Reply:
this is generated and renamed to a friendlier name in the onchainService/types